Emby.Server.Implementations/Serialization/JsonSerializer.cs

```diff
@@ -41,6 +41,27 @@ public void SerializeToStream(object obj, Stream stream)
             ServiceStack.Text.JsonSerializer.SerializeToStream(obj, obj.GetType(), stream);
         }
 
+        /// <summary>
+        /// Serializes to stream.
+        /// </summary>
+        /// <param name="obj">The obj.</param>
+        /// <param name="stream">The stream.</param>
+        /// <exception cref="ArgumentNullException">obj</exception>
+        public void SerializeToStream<T>(T obj, Stream stream)
```

Comment:
why do we need it?

Reply:
It's just a generic function instead of calling `GetType()`
I can be useful in certain situations as it gives more control of the data being serialized.